### ISSUE TITLE
add Get several league

### ISSUE THREAD

**Issue by capucine19** [api-sports/public-roadmap#75]
For example **fixtures/league/** and put many league id like : 8,28,403 ... To get all matchs from this leagues.

**Comment by zerq98** [comment]
Are there any informations when it will be done?

**Comment by Adinomis** [comment]
And/or, for example:
instead of:
https://api-football-v1.p.rapidapi.com/v3/standings?season=X&league=Y
for multi-standings in one request, use this:
https://api-football-v1.p.rapidapi.com/v3/standings_multi?standingsm(season,id)

**Comment by zerq98** [comment]
That feature will be necessary for example when you want to get detailed statistics for all teams in season in league. What exactly? When i want to display table with corners, red cards etc. For now i need to ask API 380 times for EPL.

**Comment by api-sports** [comment]
@zerq98 [zerq98](https://github.com/zerq98), you can use the parameters "ids" for this. and get multiple fixtures data like : fixtures?ids=x-x-x-x-x 

// Get fixture from severals fixtures {ids}
// In this request events, lineups, statistics fixture and players fixture are returned in the response
get("https://v3.football.api-sports.io/fixtures?ids=215662-215663-215664-215665-215666-215667");

**Comment by zerq98** [comment]
That is the response from your endpoint
https://api-football-v1.p.rapidapi.com/v3/fixtures/statistics?fixture=215662-215663-215664-215665-215666-215667

![image](https://user-images.githubusercontent.com/39220164/180493117-8afaadbe-c56d-48e2-8ee8-a9e4d7bf6ad4.png)

**Comment by api-sports** [comment]
It's the endpoint fixtures not fixtures/statistics as indicated in the documentation. But with fixtures?ids=x-x-x you get fixtures/statistics too

**Comment by zerq98** [comment]
But i can't get multiple fixture statistics like this on screen for multiple fixtures with only one ask to api
![image](https://user-images.githubusercontent.com/39220164/180494879-65691ea8-a67f-4dc3-99e0-e90c355d28f4.png)

**Comment by api-sports** [comment]
Yes, you can because these data are returned in the endpoint fixtures?ids=x-x-x as indicated in the documentation

**Comment by zerq98** [comment]
Ok sorry. Now i see this stats thanks. :D

**Comment by Phayrouz** [comment]
> That is the response from your endpoint https://api-football-v1.p.rapidapi.com/v3/fixtures/statistics?fixture=215662-215663-215664-215665-215666-215667
> 
> ![image](https://user-images.githubusercontent.com/39220164/180493117-8afaadbe-c56d-48e2-8ee8-a9e4d7bf6ad4.png)

The ids are limited to 20 fixture ids. please remove the limit just like it is done for live matches field

**Comment by Phayrouz** [comment]
> @zerq98 [zerq98](https://github.com/zerq98), you can use the parameters "ids" for this. and get multiple fixtures data like : fixtures?ids=x-x-x-x-x 
> 
> // Get fixture from severals fixtures {ids}
> // In this request events, lineups, statistics fixture and players fixture are returned in the response
> get("https://v3.football.api-sports.io/fixtures?ids=215662-215663-215664-215665-215666-215667");

And this is limited to 20 fixtures id.

**Comment by Phayrouz** [comment]
> For example **fixtures/league/** and put many league id like : 8,28,403 ... To get all matchs from this leagues.

Please, When will this be done?

**Comment by anton-ivanov-feefo** [comment]
I totally agree with @Phayrouz, if this limit is removed that would be super useful. @api-sports please consider this.

**Comment by CWilson4815** [comment]
This would be a very good addition to the API service as I currently have to call fixtures for every league in the world and then filter down to the few that I require.

**Comment by JayBox325** [comment]
Back to see if there's how long the testing process is? Very excited to come and raise this as a request to see it's being tested. 💪

**Comment by JayBox325** [comment]
It's not public. But in their roadmap, it's moved into the "testing" column.

> I haven't managed to test it, is it possible yet? On Fri, 8 Dec 2023 at 15:30, Jay ***@***.***> wrote: Back to see if there's how long the testing process is? Very excited to come and raise this as a request to see it's being tested. 💪 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***>
